client/state/ui/first-view/selectors.js

```diff
@@ -57,8 +53,7 @@ export function isViewEnabled( state, config ) {
 		return false;
 	}
 
-	const firstViewHistory = filter( getPreference( state, 'firstViewHistory' ), entry => entry.view === config.name );
-	const latestFirstViewHistory = [ ...firstViewHistory ].pop();
+	const latestFirstViewHistory = last( filter( getPreference( state, 'firstViewHistory' ), entry => entry.view === config.name ) );
```

Comment:
`findLast` is a good option here 👍
I tested the First View feature, and it is intact.